content/soarian/dstu2/individuals/patient.md

```diff
@@ -38,10 +38,10 @@ The following extensions can also be returned:
 
 	GET /Patient?:parameters
 	
-_Implementation Notes_
+### Implementation Notes
```

Comment:
This should be surrounded by underscores, not a heading. For the link above, just link to the section Implementation notes is immediately under.